tests/clone_test.go

```diff
@@ -0,0 +1,53 @@
+package tests_test
+
+import (
+	"flag"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	k8sv1 "k8s.io/api/core/v1"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/containerized-data-importer/tests"
+)
+
+const (
+	TestNS = "default"
```

Comment:
Combining this with pr #328. we should use random namespaces that are created in a `BeforeSuite` and cleaned up in an `AfterSuite` -- see related comments in that